phpstan.neon

```diff
@@ -0,0 +1,11 @@
+includes:
+    - vendor/phpstan/phpstan-doctrine/extension.neon
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+parameters:
+    level: 5
+    paths:
+        - src
+        - test
+    treatPhpDocTypesAsCertain: false
+    ignoreErrors:
+        - '#Constant Queue\\Swoole\\Command\\StartCommand::PROGRAMMATIC_CONFIG_FILES is unused#'
```

Comment:
This could be removed after removing the unused constant it is referring to.
Just make sure that `StartCommand::PROGRAMMATIC_CONFIG_FILES` is not planned to be used in the future.